.github/workflows/03-ci-lock.yml

```diff
@@ -0,0 +1,32 @@
+name: 🔒 Lock closed issues and PRs
+
+on:
+  schedule:
+    - cron: "30 2 * * *"
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  lock:
+    name: 🔐 Lock closed issues and PRs
+    runs-on: ubuntu-latest
```

Comment:
#### Change Ubuntu-latest to Ubuntu-20.04

Using the latest version may generate an error in the future

```suggestion
    runs-on: ubuntu-20.04
```